mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-examples.adoc

```diff
@@ -0,0 +1,22 @@
+= MQTT3 Connector Examples - Mule 4
+
+To fully benefit from using Anypoint Connector for MQTT3 (MQTT3 Connector), see the following examples of configuring the connector:
```

Comment:
Here are some example showing how to configure Anypoint Connector for MQTT3 (MQTT3 Connector):

Reply:
I left the previous text and updated the name, as it's the standard text worked with our Docs Editor.